plugins/bnet_plugin/bnet_plugin.cpp

```diff
@@ -1403,7 +1384,6 @@ namespace eosio {
 
       if( app().get_plugin<chain_plugin>().chain().get_read_mode() == chain::db_read_mode::READ_ONLY ) {
          my->_request_trx = false;
-         ilog( "setting bnet-no-trx to true since in read-only mode" );
```

Comment:
I would prefer to keep the ilog so it is clear the side effect. You can ilog only if my->_request_trx is true.

Reply:
ok, added ilog when bnet-no-trx=false